components/core/header/nav-bar/NavBarHamburger.vue

```diff
@@ -82,6 +82,7 @@
                 @click.native="toggleAccordion('venue')"
             ></nav-bar-item-accordion>
             <ext-link
+                v-if="showProposalSystem"
```

Comment:
Is this working???
due to you're using showProposalSystem flag,
we should add this property in computed.
something like this
```js
showProposalSystem() {
            return this.$store.state.configs.showProposalSystem
        },
```

Reply:
> Is this working??? due to you're using showProposalSystem flag, we should add this property in computed. something like this
> 
> ```js
> showProposalSystem() {
>             return this.$store.state.configs.showProposalSystem
>         },
> ```

Thanks for reviewing! I'm uploading it on staging and checking(my local deploy is not working currently

Reply:
@baby230211 It does work. 
<img width="1101" alt="PyCon Taiwan" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/6ad49d68-de5a-4c52-81cd-0653b3f55939">
I learnt it from switches in store/index.js. Dug in a bit, it is started from a helpful commit from Josix: https://github.com/pycontw/pycontw-frontend/commit/3c2d0429c4f99e15cb7a56b30e77da91eadfc48f